### PR TITLE
Update community_tracers_instrumentation.yml

### PR DESCRIPTION
**Pull request by guoqqqi**
add Apache APISIX plugin Zipkin for the Community supported

### CODE REVIEW

File: _data/community_tracers_instrumentation.yml [hunk 1]
```diff
@@ -167,6 +167,15 @@
...
   notes: A [Kong](http://konghq.com/) plugin to enable tracing to a zipkin server.
 
+- language: Lua
+  library: >-
+    [Apache APISIX-plugin-zipkin](https://github.com/apache/apisix/tree/master/apisix/plugins/zipkin)
+  framework: [Apache APISIX](https://apisix.apache.org/)
+  propagation: Http (B3)
+  transports: Http
+  sampling: "Yes"
+  notes: A [Apache APISIX](https://apisix.apache.org/) plugin to enable tracing to a zipkin server.
```
Comment (tao12345666333) [inline _data/community_tracers_instrumentation.yml:177]:
```suggestion
  notes: An [Apache APISIX](https://apisix.apache.org/) plugin to enable tracing to a zipkin server.
```

**Review by jcchavezs — APPROVED** [review]
Thank you!

**Comment by guoqqqi** [comment]
Hi, @jcchavezs Do you have time to re-trigger the CI for me? 😄

**Comment by jcchavezs** [comment]
Thank you.

**Pull request merged**